src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.model;
+
+import java.util.NoSuchElementException;

```

Comment:
It seems like you've imported NoSuchElementException, but it is not being used in this file. If you are not using it elsewhere in your code, consider removing this import to keep your code clean.